packages/backend/src/controllers/authentication/auth.ts

```diff
@@ -10,22 +10,22 @@ import { AuthToken } from './token.schema';
 import jwt from 'jsonwebtoken';
 import expressjwt from 'express-jwt';
 import { config } from '../../configuration/environment';
-import { ApiError } from '../error/api.class';
 import { sendTo } from '../discord';
 import { TokenRequestMessage } from '../discord/templates';
+import { ApiError, ApiSuccess } from '../../utils/api';
 
-const authTokenReqeuestSchema = object({
-  username: string().required(),
+const authTokenRequestSchema = object({
+  username: string().required().regex(/[\w\s]+#[0-9]+/).description('Discord username#0000 of the admin'),
 });
 
 const authLoginRequestSchema = object({
-  username: string().required(),
+  username: string().required().regex(/[\w\s]+#[0-9]+/).description('Discord username#0000 of the admin'),
```

Comment:
Same regex as in administrator/index.ts?
```suggestion
  username: string().required().regex(/^[\w\s]+#\d{4}$/).description('Discord username#0000 of the admin'),
```

Reply:
Habe hierfür eine neues Ticket #64  aufgemacht, damit ich das einfach mergen kann